src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConstructorSnippetProvider.cs

```diff
@@ -34,9 +34,12 @@ protected override async Task<TextChange> GenerateSnippetTextChangeAsync(Documen
             var nodeAtPosition = root.FindNode(TextSpan.FromBounds(position, position));
             var containingType = nodeAtPosition.FirstAncestorOrSelf<SyntaxNode>(syntaxFacts.IsTypeDeclaration);
             Contract.ThrowIfNull(containingType);
+            var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+            var containingTypeSymbol = semanticModel.GetDeclaredSymbol(containingType, cancellationToken);
+            Contract.ThrowIfNull(containingTypeSymbol);
```

Comment:
`containingType` is guaranteed to be a type declaration node, so `containingTypeSymbol` must never be `null`